DEV_README.md

```diff
@@ -83,8 +82,30 @@ in to this you need to add the following to your plugin.xml
     </extension>
     
     
-Step 4 - You're done!
----------------------
+### Step 4 - You're done!
+
 
 Error markers will now appear in the right places. More work can be done to add quick fixes, but
 it's much harder than adding the markers.
+
+Build against local Bnd-Lib
+----------------------------
+
+Since BndTools relies on Bnd, it is sometimes necessary to use the local version of Bnd. This dependency is configured in `cnf/build.bnd` with a declaration of `bndlib` which is referenced in all projects.
+
+	bndlib:		aQute.libg;version='[${bnd-version-base},${bnd-version-ceiling})', \
+                biz.aQute.bndlib;version='[${bnd-version-base},${bnd-version-ceiling})'
+
+### Eclipse
+
+In order to configure Eclipse, change bndlib to reference the file generated in your Bnd-Workspace
+
+	bndlib		<YOUR SYSTEM PATH>/bnd/biz.aQute.bndlib/generated/biz.aQute.bndlib.jar;version=file
```

Comment:
I updated the PR, though I kept Peters solution as a alternative since you can skip the full release-build.